docs/react-v9/contributing/migration-notes.md

```diff
@@ -0,0 +1,407 @@
+# Migration notes from `@fluentui/react-components@9.0.0-beta.5`to `@fluentui/react-components@9.0.0-rc.1`
+
+## Changes to the styling system
+
+### Functions no longer supported
+
+Functions in `makeStyles()` are no longer supported, the `tokens` object can be used directly instead.
+
+Please apply following changes:
+
+```diff
+-import { makeStyles } from '@fluentui/react-components';
++import { makeStyles, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+-  root: theme => ({ color: theme.tokenB }),
++  root: { color: tokens.tokenB },
+});
+```
+
+Focus indicator style functions in `@fluentui/react-tabster` were also updated to support this change:
+
+```diff
+-import { createCustomFocusIndicatorStyle, createFocusOutlineStyle, makeStyles } from '@fluentui/react-components';
++import { createCustomFocusIndicatorStyle, createFocusOutlineStyle, makeStyles, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+-  focusOutline1: createFocusOutlineStyle(theme, { selector: 'focus-within', style: { outlineOffset: '8px' } }),
++  focusOutline1: createFocusOutlineStyle({ selector: 'focus-within', style: { outlineOffset: '8px' } }),
+
+-  focusOutline2: createFocusOutlineStyle(theme => ({ backgroundColor: theme.colorNeutralBackground1 })),
++  focusOutline2: createFocusOutlineStyle({ backgroundColor: tokens.colorNeutralBackground1 }),
+});
+```
+
+For more details, please check [microsoft/fluentui#20651](https://github.com/microsoft/fluentui/pull/20651).
+
+### CSS shorthands no longer supported
+
+[CSS shorthands](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties) in `makeStyles()` calls are no longer supported. For many shorthands there exist matching functions in `@fluentui/react-components`:
```

Comment:
This isn't really accurate anymore right? https://griffel.js.org/react/guides/limitations#some-of-css-shorthands-are-not-supported

I think we should update this.